experimental/terraform/resource_event_handler.tf

```diff
@@ -118,6 +118,12 @@ resource "google_secret_manager_secret_iam_member" "event_handler" {
   member    = "serviceAccount:${google_service_account.fourkeys.email}"
 }
 
+resource "google_secret_manager_secret_iam_member" "pager_duty" {
```

Comment:
I think there are two parts to this looking at the event_handler above, also can we combine these two to loop through the secrets.

Reply:
@matthew-green-nandos , For event_handler secret .. it was created using the same terraform with random value and we think the actual value was inserted using "gcloud command" on top of it. So correct me if i am wrong.. if we would loop through  (_for secret creation, versioning and permission so it can support pagerduty and event_handler from same code_)  it would recreate the secret for event_handler and we might lose existing event_handler github token value which is getting used as git hub token.  Or i can just do secret access permissions as we already have pager_duty secret in the secret manager.

Reply:
I have the secret and no problem recreating it. Can we not do a `terraform state mv` to refactor the code?

Reply:
Since we are not able to obtain the Pager Duty webhook subscription secret via Terraform, I think literally all we're interested in here is adding IAM bindings to give our service account access to the Pager Duty secret. Granting access in a loop is fine and won't require any manual state adjustments (it will just remove and then immediately grant permission to the `event_handler` secret (with the additional access for our `pager_duty_secret`).

Assuming that makes sense, the only question I have is if you're happy with the `terraform plan` [here](https://console.cloud.google.com/cloud-build/builds;region=global/c1fd9aac-38fd-4838-aa5f-1e573475da8a?authuser=1&project=mgt-fourkeys-8d87d44d) (specifically L389):

```
Step #3 - "tf plan":   # google_secret_manager_secret.event_handler will be updated in-place
Step #3 - "tf plan":   ~ resource "google_secret_manager_secret" "event_handler" {
Step #3 - "tf plan":         id          = "projects/mgt-fourkeys-8d87d44d/secrets/event-handler"
Step #3 - "tf plan":         name        = "projects/967866933655/secrets/event-handler"
Step #3 - "tf plan":         # (4 unchanged attributes hidden)
Step #3 - "tf plan": 
Step #3 - "tf plan":       ~ replication {
Step #3 - "tf plan":             # (1 unchanged attribute hidden)
Step #3 - "tf plan": 
Step #3 - "tf plan":           ~ user_managed {
Step #3 - "tf plan":               ~ replicas {
Step #3 - "tf plan":                   ~ location = "europe-west2" -> "europe-west1"       # <-- L389
Step #3 - "tf plan":                 }
Step #3 - "tf plan":             }
Step #3 - "tf plan":         }
Step #3 - "tf plan":     }
```

It appears to be moving the replica to a different region, based on the input of `_FOURKEYS_DOMAIN_MAPPING_REGION`.